pkg/cloud/cloud.go

```diff
@@ -169,6 +169,7 @@ func (c *cloud) ResizeFileSystem(ctx context.Context, fileSystemId string, newSi
 				if err != nil {
 					return nil, err
 				}
+				return &newSizeGiB, nil
```

Comment:
Note, @hughdanliu and I discussed making this a separate commit/PR offline. Is this causing the sanity tests to fail @gomesjason? If not, I'm inclined to believe this needs an additional sanity test to catch it, if so, can you break this into a separate commit for cleanliness sake

Reply:
correction, @hughdanliu pointed out that the sanity tests wouldn't be expected to cover this. Would it be possible to break it into its own commit and add a corresponding unit test

Reply:
Yea, added it since it breaks a unit test but we can move it elsewhere

Reply:
There was an existing unit test to address this situation but it was broken on release. Just for the sake of velocity this particular bug fix seems fine to leave it in here, but if @jacobwolfaws feels otherwise I don't feel strongly about this.

Reply:
I'd prefer it if you staggered the commits, the first being the bug fix and the second being the sanity tests. It's cleaner that way imo

Reply:
Fixed